src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -361,9 +363,10 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
       for index, ppa in enumerate(config['ppaRecords']):
         del ppa['id']
         exist_in_dump = False
-        for ppa_record in ppa_dump_data:			           
+        for ppa_record in ppa_dump_data:
```

Comment:
Better to put it just after line 357, no? Then you avoid the "nested `for` loop" problem and do the check the minimum number of times.

Reply:
Acually I found it more clear to seperate the loops , I updated the code removing one loop for each of the ppa and the esc sensor, but I think we need the nested loop anyway, right?

Reply:
i didn't get your opinion if you agree that we comment the GeoJSON winding checks and add TODO comment, and we can re add it after verifying with WG1 if they will keep this winding constraints.